.circleci/config.yml

```diff
@@ -3,50 +3,17 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/python:3.7.0-stretch-browsers
+    - image: iodide/pyodide:0.2.0
```

Comment:
Maybe we should then increment the `__version__` in `pyodide_build` at the same time? https://github.com/iodide-project/pyodide/blob/01256b5832e087d94e30559fee3034001ce511b5/pyodide_build/__init__.py#L1

Reply:
I had sort of seen these as independent versions (until such time as, perhaps, we distribute the docker image as a way of getting a ready-to-go pyodide_build environment for building custom packages).  But maybe I'm missing a reason for this that you had in mind?

Reply:
True, but it may be somewhat confusing to users. When I see a docker image `pyodide:0.2.0` the natural assumption is that docker image does contain pyodide version 0.2.0, while currently it's not the case. Not only it doesn't contain pyodide but the version number is completely unrelated.

Also if we do decide to distribute pre-build docker images to facilitate building of custom packages in the future (which may be important to reduce the usage complexity barrier) how will we ensure consistency between version. Say then we will get pyodide v0.4 and this image v0.12, wondering what should happen then..

Maybe it could be worth renaming this image `pyodide-env` (or something similar) instead to avoid confusion? If not, I'm fine merging it as it is and postpone this discussion to when we actually run into that issue..